Meadowlark-js/services/meadowlark-fastify/src/Factory.ts

```diff
@@ -11,9 +11,24 @@ export type ServiceFactory = (worker: number) => Promise<void>;
 /* istanbul ignore file */
 export async function serviceFactory(worker: number) {
   const service: FastifyInstance = buildService();
+
+  const closeGracefully = async (signal: string | number | undefined) => {
+    Logger.info(`Received signal to terminate: ${signal}`, null);
+
+    await service.close();
+    // TODO: close database connections Initial thoughts: may need to have a global registry of backends to go through,
+    // manually closing any connections.
```

Comment:
I should create a separate ticket for dealing with this issue. While reading about Docker best practices, I came across a comment like "don't forget to close your database connections." That's a great point.

Alternately, might go into `Services.ts` below. Need to experiment in Windows and inside of Linux (container) to see what makes the most sense.